core/src/main/resources/application.conf

```diff
@@ -0,0 +1,106 @@
+collector {
+  enableDefaultRedirect = false
+
+  paths {
+
+  }
+
+  crossDomain {
+    enabled = false
+    domains = [ "*" ]
+    secure = true
+  }
+
+  cookieBounce {
+    enabled = false
+    name = "n3pc"
+    fallbackNetworkUserId = "00000000-0000-4000-A000-000000000000"
+  }
+
+  cookie {
+    enabled = true
+    expiration = 365 days
+    name = sp
+    domains = [
+
+    ]
+    fallbackDomain = "acme1.com"
+    secure = true
+    httpOnly = false
+    sameSite = "None"
+  }
+
+  doNotTrackCookie {
+    enabled = false
+    name = ""
+    value = ""
+  }
+
+  ssl {
+    enable = false
+    redirect = false
+    port = 9543
+  }
+
+  p3p {
+    policyRef = "/w3c/p3p.xml"
+    CP = "NOI DSP COR NID PSA OUR IND COM NAV STA"
+  }
+
+  prometheusMetrics {
+    enabled = false
+  }
+
+  rootResponse {
+    enabled = false
+    statusCode = 302
+    headers = {}
+    body = "302, redirecting"
+  }
+
+
+  redirectMacro {
+    enabled = false
+  }
+
+  cors {
+    accessControlMaxAge = 5 seconds
```

Comment:
@paulboocock could also comment on this one.

Reply:
I'm not sure why this is so short to be honest, besides 5 seconds being the chromium default. Increasing this would lead to fewer OPTIONS requests being sent and given that a preflight from the collector would be unlikely the change, I wonder if a large value like `600` (10 minutes) would be better. I picked `600` because thats the cap for Chromium < 76. 

Another benefit of increasing this value is that we increase the chance of events being captured on page navigation, since we're not waiting for the OPTIONS request before sending the POST.

Downsides would be having to wait 10 minutes for it to expire if it's misconfigured or if the configuration changes, which could lead to CORS errors.